packages/web3-eth/test/unit/detect_transaction_type.test.ts

```diff
@@ -0,0 +1,103 @@
+﻿/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+import { InvalidPropertiesForTransactionTypeError } from 'web3-errors';
+import { Web3Context } from 'web3-core';
+import { EthExecutionAPI } from 'web3-types';
+import HttpProvider from 'web3-providers-http';
+
+import { detectTransactionType } from '../../src/utils/detect_transaction_type';
+import {
+	transactionType0x0,
+	transactionType0x1,
+	transactionType0x2,
+	transactionTypeUndefined,
+	transactionTypeValidationError,
+} from '../fixtures/detect_transaction_type';
+import { getBlock } from '../../src/rpc_method_wrappers';
+
+describe('detectTransactionType', () => {
+	beforeAll(() => {
+		
+		jest.mock('../../src/rpc_method_wrappers');
+		// eslint-disable-next-line
+		(getBlock as jest.Mock).mockReturnValue(() => {baseFeePerGas: 1});
+	});
+	afterAll(() => {
+		jest.resetAllMocks();
+	});
+	describe('should override detectTransactionType method', () => {
+		const web3Context = new Web3Context<EthExecutionAPI>({
+			provider: new HttpProvider('http://127.0.0.1:80'),
+		});
+		it.skip('should call override method', async () => {
```

Comment:
why this test is skipped?

Reply:
this file is just a rename and is an existing testcase, i dont think it was completed so i think i can remove it